data/events.yml

```
@@ -10464,3 +10464,38 @@
       implemented under the hood by customizing LiveData. While the talk
       discusses this specific framework, the ideas in it are universal and
       can be used independently in any project.</p>
+- lang: en
+  startDate: '2022-07-13'
+  endDate: '2022-07-13'
+  location: Lisbon, Portugal
+  speaker: Carlos Mota, Android GDE
+  title: Kotlin Lisboa Meetup #8
```

Please use quotes here, because now "#" starts a comment line.

```suggestion
  title: 'Kotlin Lisboa Meetup #8'
```